beater/journalbeat.go

```diff
@@ -288,6 +288,20 @@ func (jb *Journalbeat) Run(b *beat.Beat) error {
 
 		event["input_type"] = jb.config.DefaultType
 		event["cursor"] = rawEvent.Cursor
+		if tmStr, ok := rawEvent.Fields[timestampField]; ok {
+			tm, err := strconv.ParseInt(tmStr, 10, 64)
+			if err == nil {
+				tmSecs := tm / microseconds
+				tmUSecs := tm % microseconds
+				tm := time.Unix(tmSecs, tmUSecs*microsToNanos)
+				event["@timestamp"] = tm.Format("2006-01-02T15:04:05.760738998Z")
+			}
```

Comment:
In this case logstash aggregator will stamp a time on the server side to the message. This is one of the cases which should never happen based on the data I have looked at so far.